packages/backend/src/auth/jwttoken.dto.ts

```diff
@@ -0,0 +1,16 @@
+import {IJWTTokenRequest} from '@project/common/features/auth/models';
+import {ApiProperty} from '@nestjs/swagger';
+export class JWTTokenDTO implements IJWTTokenRequest {
```

Comment:
DTO to pattern do komunikacji z zewnetrznymi tworami nie musi sie tak nazywac kady model